description.md

```diff
@@ -3,11 +3,44 @@
 This file contains all the topics that are covered by the Open Education Hub methodology.
 Each topic in this file contains a small description and links to the files cover it more thouroughly.
 
-
 ## Teaching Stages
 
 ### Lectures
 
+Lectures are teaching activities that typically take place with a larger audience and are asymmetrical: a lecturer mostly talks while the audience mostly listens.
+Interaction is (quite obviously) important, but it won't scale to a larger audience and, because the entire communication medium is shared, is limited to on conversation only.
+Labs, for example, allow each participant to focus on one item they could work on.
+Seminars are close to lectures, except that they typically involve a smaller group of people and are naturally interactive, with the relationship being less asymmetrical.
+
+Materials are required for delivering lectures.
+In the following sections, we detail steps in preparing and (re)using lecture materials in Open Education Hub.
+
+We organize lecture materials according lecture-specific activities:
+
+* **Visual support materials** are used as support material for giving talks and engaging participants in discussions.
+  They consist of:
+  * **Slides** are the typical structured and simplified format for organizing ideas.
+    The lecturer typically walks through slides and presents information by expanding ideas on the slides, and by engaging in discussions based on those ideas.
+  * **Media files** are generally images or videos that represent a concept in an easier to grasp format.
```

Comment:
I don't think this category is necessary. You can say that media and reading are embedded into slides.

Reply:
Maybe some media may not be embedded. I think I can leave it separate, but mention that media is typically embedded into slides.